terraform/secrets.tf

```diff
@@ -16,13 +16,14 @@ resource "aws_secretsmanager_secret_version" "gov_notify_api_key" {
   secret_string = var.gov_notify_api_key
 }
 
-resource "aws_secretsmanager_secret" "gov_notify_customer_email_template" {
-  name = "${terraform.workspace}_gov_notify_customer_email_template"
+resource "aws_secretsmanager_secret" "gov_notify_feedback_email_address" {
+  name = "${terraform.workspace}_gov_notify_feedback_email_address"
+  tags = module.beacons_label.tags
 }
 
-resource "aws_secretsmanager_secret_version" "gov_notify_customer_email_template" {
-  secret_id     = aws_secretsmanager_secret.gov_notify_customer_email_template.id
-  secret_string = var.gov_notify_customer_email_template
+resource "aws_secretsmanager_secret_version" "gov_notify_feedback_email_address" {
+  secret_id     = aws_secretsmanager_secret.gov_notify_feedback_email_address.id
+  secret_string = var.gov_notify_feedback_email_address
 }
```

Comment:
Again, doesn't need to be managed in AWS SecretsManager because not secrets.  Slow changing, so hard-coded in Webapp's `GovNotifyGateway`.